packages/ag-grid-theme/css/_salt-ag-theme-mixin.scss

```diff
@@ -183,6 +183,7 @@
     background-color: var(--agGrid-row-background-secondary);
   }
 
+  .ag-row-selected.ag-row-hover::before,
```

Comment:
This needs a changeset

Reply:
here goes: https://github.com/jpmorganchase/salt-ds/pull/1018